webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+  Label,
+} from '@patternfly/react-core';
+
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import ContentViewIcon from '../../../../scenes/ContentViews/components/ContentViewIcon';
+
+
+const ContentViewDetailsCard = ({ hostDetails }) => {
+  let contentViewDetails;
+  if (hostDetails.content_facet_attributes) {
+    const cv = hostDetails.content_facet_attributes.content_view;
+    const env = hostDetails.content_facet_attributes.lifecycle_environment;
+    const versionId = hostDetails.content_facet_attributes.content_view_version_id;
+    const version = hostDetails.content_facet_attributes.content_view_version;
```

Comment:
Destructuring in combo with the `propsToCamelCase` helper can help quite a bit here:
```suggestion
  import { propsToCamelCase } from 'foremanReact/common/helpers';
  ...
  const hostDetailsData = propsToCamelCase(hostDetails.content_facet_attributes);
  const { contentView, lifecycleEnvironment, contentViewVersionId, contentViewVersion } = hostDetailsData;
  
```